apps/web/src/app/globals.css

```
@@ -475,8 +477,8 @@
 /* Scoped to #cc-main so specificity beats the library's :root defaults
    regardless of CSS chunk load order (library CSS loads dynamically). */
 #cc-main {
-  --cc-font-family: "Montserrat", "-apple-system", system-ui, "BlinkMacSystemFont",
-    "Segoe UI", "Roboto", "Helvetica Neue", "Arial", sans-serif;
+  --cc-font-family: var(--font-montserrat), -apple-system, system-ui, BlinkMacSystemFont,
```

Stylelint (17.6.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)